flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -247,6 +265,27 @@ int inferParallelism(FlinkInputFormat format, ScanContext context) {
       parallelism = Math.max(1, parallelism);
       return parallelism;
     }
+
+    boolean localityEnabled() {
+      Boolean localityConfig =
+          this.exposeLocality != null ? this.exposeLocality :
+              readableConfig.get(FlinkConfigOptions.TABLE_EXEC_ICEBERG_EXPOSE_SPLIT_LOCALITY_INFO);
+
+      if (localityConfig != null && !localityConfig) {
```

Comment:
It would be more clear when checking a variable if it has a name like `localityEnabled`. That is still an appropriate name and it makes this logic easier to read.